src/utils/viewer/viewer.cpp

```diff
@@ -352,14 +373,27 @@ void Viewer::drawEvent() {
     physicsManager_->debugDraw(projM * camM);
   }
 
+  if (displayPerformanceInfo) {
+    Mn::Color3 textColor{1.0f};
+    textRenderer_->draw(textColor);
```

Comment:
`textRenderer_->draw(0xffffff_rgbf)`, since we have the hexadecimal / CSS color literals?

Reply:
OK, will change.